staging/src/k8s.io/apiserver/pkg/server/deleted_kinds.go

```diff
@@ -118,6 +118,9 @@ func (e *resourceExpirationEvaluator) shouldServe(gv schema.GroupVersion, versio
 }
 
 func (e *resourceExpirationEvaluator) ShouldServeForVersion(majorRemoved, minorRemoved int) bool {
+	if minorRemoved == 22 {
```

Comment:
change to `if e.currentMinor == 22 && minorRemoved == 22` if you want to mimic the logic that will apply once 1.22 reaches beta